dist/profile/templates/kubernetes/resources/repo_proxy/deployment.yaml.erb

```diff
@@ -16,30 +16,36 @@ spec:
                 type: repo-proxy
                 logtype: archive
         spec:
+            # Nginx user need write permission on /var/cache/nginx
+            initContainers:
+                - name: volume-mount-hack
+                  image: busybox
+                  command: ["sh", "-c", "chown -R 101:101 /var/cache/nginx"]
```

Comment:
Is the nginx user guaranteed to always be 101 in the container? Assuming not, this could be something like `$(id -u nginx):$(id -g nginx)`